src/config/rcxml.c

```diff
@@ -895,9 +895,13 @@ entry(xmlNode *node, char *nodename, char *content)
 	} else if (!strcasecmp(nodename, "outlines.windowSwitcher")) {
 		set_bool(content, &rc.window_switcher.outlines);
 	} else if (!strcasecmp(nodename, "allWorkspaces.windowSwitcher")) {
+		set_bool(content, &rc.window_switcher.allworkspaces);
 		if (parse_bool(content, -1) == true) {
 			rc.window_switcher.criteria &=
 				~LAB_VIEW_CRITERIA_CURRENT_WORKSPACE;
+		} else {
+			rc.window_switcher.criteria |=
+				LAB_VIEW_CRITERIA_CURRENT_WORKSPACE;
```

Comment:
This shouldn't be needed as it is initialized that way already.

Reply:
I wasn't sure about the initialization, thus the else and whether it got reset on a reconfig.

Reply:
`rcxml_init()` is called on every reconfigure.